token/cli/src/main.rs

```diff
@@ -1377,13 +1377,22 @@ fn command_close(
         }
     }
 
-    let instructions = vec![close_account(
+    let mut instructions = vec![close_account(
         &config.program_id,
         &account,
         &recipient,
         &close_authority,
         &config.multisigner_pubkeys,
     )?];
+
+    let recipient_account = config
+        .rpc_client
+        .get_token_account(&recipient)?;
+
+    if !recipient_account.is_none() && recipient_account.unwrap().is_native {
```

Comment:
Be sure to run clippy to do this check in a clearer way with: `$ cargo +nightly-2022-02-24 clippy -Zunstable-options --all-targets`